libs/langchain-community/src/document_loaders/tests/assemblyai.int.test.ts

```diff
@@ -6,10 +6,9 @@ import {
   AudioTranscriptLoader,
   AudioTranscriptParagraphsLoader,
   AudioTranscriptSentencesLoader,
-} from "../web/assemblyai.js";
+} from "../web/assemblyai.ts";
```

Comment:
This should not change
```suggestion
} from "../web/assemblyai.js";
```